pkg/openfeature/logger.go

```diff
@@ -21,7 +21,9 @@ func (l logger) Enabled(level int) bool { return true }
 
 func (l logger) Info(level int, msg string, keysAndValues ...interface{}) {}
 
-func (l logger) Error(err error, msg string, keysAndValues ...interface{}) { log.Println(err) }
+func (l logger) Error(err error, msg string, keysAndValues ...interface{}) {
+	log.Println("openfeature:", err)
```

Comment:
It looks like consensus has been that we'll default to no op even in error cases so I think this line can be removed entirely.

Reply:
I think this should be done in a separate PR. That way, we can weigh the pros/cons of not logging anything out of the box.